### PR TITLE
Dockerize

### PR DESCRIPTION
**Pull request by kovacshuni**

### CODE REVIEW

File: app.go [hunk 1]
```diff
@@ -369,9 +383,21 @@ func (bn brightcoveNotifier) prettyPrint() string {
 }
 
 func (bc brightcoveConfig) prettyPrint() string {
-	return fmt.Sprintf("\n\t\taddr: [%s]\n\t\toauthAddr: [%s]\n\t\taccountID: [%s]\n\t", bc.addr, bc.oauthAddr, bc.accountID)
```
Comment (catalin-ilea) [inline app.go:372]: Can you please make sure you don't log sensitive information like tokens and auth details? Otherwise they'll end up in splunk.

**Comment by Nesze** [comment]
some redundant ~~params~~ paranthesis in `if` statements, otherwise 👍 

Edit: please run `go fmt`

File: app.go (hunk 1 above)
Comment (kovacshuni) [inline app.go:372]: No sensitive info is logged.

**Comment by BerniVarga** [comment]
+1

**Pull request merged**